src/bleanser/core/processor.py

```diff
@@ -22,8 +23,44 @@
 from plumbum import local # type: ignore
 
 
+# Mixin for helpers, this lets user subclass nicer
+class NormaliserHelpers:
+    @staticmethod
+    def unique_file_in_tempdir(unpacked_filepath: Path, wdir: Path, *, suffix: Optional[str] = None) -> Path:
```

Comment:
nit: best to make keyword only since they are both of the same type and easy to swap by accident